homeassistant/components/satel_integra/manifest.json

```diff
@@ -1,10 +1,12 @@
 {
   "domain": "satel_integra",
   "name": "Satel Integra",
-  "codeowners": [],
+  "codeowners": ["@Tommatheussen"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/satel_integra",
   "iot_class": "local_push",
   "loggers": ["satel_integra"],
   "quality_scale": "legacy",
-  "requirements": ["satel-integra==0.3.7"]
+  "requirements": ["satel-integra==0.3.7"],
+  "single_config_entry": true
```

Comment:
Why only a single config entry though?

Reply:
Right, I didn't take the unique ids into account. I think it would be a nice option to allow multiple entries, but that indeed comes with that we have to migrate existing unique ids, which isn't bad, but I think we could do that in a followup PR to make this PR smaller. So I am fine to keep it out for now